modules/basis_universal/register_types.cpp

```diff
@@ -190,9 +190,6 @@ static Ref<Image> basis_universal_unpacker_ptr(const uint8_t *p_data, int p_size
 			} else if (RS::get_singleton()->has_os_feature("s3tc")) {
 				format = basist::transcoder_texture_format::cTFBC1; // get this from renderer
 				imgfmt = Image::FORMAT_DXT1;
-			} else if (RS::get_singleton()->has_os_feature("etc")) {
```

Comment:
Note that this will break BasisU transcoding for opaque images on mobile, this could probably be changed to something like:
```cpp
else if (RS::get_singleton()->has_os_feature("etc2")) {
	format = basist::transcoder_texture_format::cTFETC1; // get this from renderer
	imgfmt = Image::FORMAT_ETC2_RGB8;
}
```

In theory this should work correctly since BasisU transcodes ETC2 RGBA as EAC alpha + ETC1 block anyways, though it would still need testing on a mobile device.

Reply:
Ah indeed, I forgot that this check I changed in the renderer code means both `etc` and `etc2` are defined and synonymous:
```cpp
if (p_feature == "etc" || p_feature == "etc2") {
	return config->etc2_supported;
}
```

For this PR, I could just replace `etc` by `etc2` and keep the format as is, which would in theory be equivalent to the status quo, and then actually changing the format could be evaluated in a follow-up?

Or I'm happy to try your version here, but indeed we need to confirm it works well.

Reply:
Updated with your snippet, thanks!